src/test/java/leetcode/longestCommonPrefix/PrefixTest.java

```diff
@@ -0,0 +1,25 @@
+package leetcode.longestCommonPrefix;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PrefixTest {
+    @Test
+    public void shouldReturnPrefix() {
+        String[] array1 = {"flower", "flow", "flight"};
```

Comment:
A. Rozbiłbym to na testy:
1. shouldReturrnPrefixWhenWholeWordIsAPrefix 
2. shouldReturnPrefixWhenPartOfWordIsAPrefix
3. shouldReturnEmptyStringWhenNoPrefix
4. shouldReturnEmptyStringWhenEmptyInput

B. Używaj sekcji //given //when //then kiedy piszesz tetsy